### PR TITLE
Issue #349: $expand contain $count don't work

### PR DESCRIPTION
**Pull request by xuzhg**
Issue #349: $expand contain $count don't work

### CODE REVIEW

File: OData/test/UnitTest/System.Web.OData.Test/OData/SelectExpandNestedDollarCountTest.cs [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+            // Act
+            HttpResponseMessage response = GetResponse(uri, AcceptJson);
+
+            // Assert
+            Assert.NotNull(response);
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+
+            string payload = response.Content.ReadAsStringAsync().Result;
+            Assert.Contains("\"Orders@odata.count\":5,", payload); // Top
```
Comment (xuzhg) [inline OData/test/UnitTest/System.Web.OData.Test/OData/SelectExpandNestedDollarCountTest.cs]: need to change the comment.

File: OData/test/UnitTest/System.Web.OData.Test/OData/SelectExpandNestedDollarCountTest.cs [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+                    typeof(MsCustomersController), typeof(MetadataController)
+                }.GetHttpConfiguration();
+            _configuration.IncludeErrorDetailPolicy = IncludeErrorDetailPolicy.Always;
+
+            _configuration.MapODataServiceRoute("odata", "odata", GetModel());
+            HttpServer server = new HttpServer(_configuration);
+            _client = new HttpClient(server);
+        }
+
+        [Theory]
```
Comment (xuzhg) [inline OData/test/UnitTest/System.Web.OData.Test/OData/SelectExpandNestedDollarCountTest.cs]: not necessary.

**Comment by xuzhg** [comment]
Merged. Thanks.

**Comment by VikingsFan** [comment]
Merged 7255614b4cb65576c8e40784f9d51b27be5d8fd3

**Pull request closed without merging**